src/js/iov/player.js

```diff
@@ -444,12 +450,33 @@ export default class IOVPlayer {
       // this.iov.conduit.disconnect();
     }
 
-    // Don't wait until the next play event or the destruction of this player
-    // to clear the MSE
-    if (this.mseWrapper) {
-      await this.mseWrapper.destroy();
-      this.mseWrapper = null;
-    }
+    debug('stop about to finish synchronously...');
```

Comment:
This should state 'stop about to finish synchronous operations and return promise' - again for clarity on what is going on.